src/components/register_student/index.js

```diff
@@ -562,6 +580,26 @@ class RegisterStudent extends React.Component {
                   />
                 </Form.Field>
               </Form.Group>
+              <Form.Group fluid widths='equal'>
+                <Form.Field>
+                  <label>"Address (As Per Bhawan)"</label>
+                  <Input
+                    name='address_bhawan'
+                    value={address_bhawan}
+                    onChange={this.fieldsChange}
+                    loading={loading}
+                  />
+                </Form.Field>
+                <Form.Field fluid widths='equal'>
```

Comment:
```suggestion
                <Form.Field>
```

@Hardik-2311 I think this should be changed?

Reply:
apne hi kaha tha input field match krni hai bkiyo se vohi hai yeh

Reply:
it's required for `Form.Group`, not for `Form.Field` i think.